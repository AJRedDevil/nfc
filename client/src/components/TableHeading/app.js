import React from 'react';
import {string} from 'prop-types';

const TableHeader = ({title, subtitle}) => (
  <section className="hero is-small is-info is-bold">
    <div className="hero-body">
      <div className="container">
        <h1 className="title">{title}</h1>
        <h2 className="subtitle">{subtitle}</h2>
      </div>
    </div>
  </section>
);
TableHeader.propTypes = {
  title: string.isRequired,
  subtitle: string,
};

export default TableHeader;
