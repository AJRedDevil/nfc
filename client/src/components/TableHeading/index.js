import TableHeader from './app';

export default TableHeader;
