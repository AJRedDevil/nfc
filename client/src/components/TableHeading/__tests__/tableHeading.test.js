// npm packages
import React from 'react';
import renderer from 'react-test-renderer';

// our packages
import TableHeading from '../index';

test('# TableHeading', () => {
  const wrapper = renderer
    .create(<TableHeading title="test" subtitle="snapshot" />)
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
