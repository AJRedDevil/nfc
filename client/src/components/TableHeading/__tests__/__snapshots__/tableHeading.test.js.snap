// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# TableHeading 1`] = `
<section
  className="hero is-small is-info is-bold"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        test
      </h1>
      <h2
        className="subtitle"
      />
    </div>
  </div>
</section>
`;
